Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+importlib_metadata = "*"
+rich = "*"
+requests = "*"
+pandas = "*"
+ipython = "*"
```

Comment:
might need to pin these in a future version

Reply:
Yes, perhaps in the setup.py `install_requires` list rather than the Pipfile, since it can act as the "bleeding edge" that runs against the tests between releases.

Reply:
Good idea, I'll update that.

Reply:
Seems like this is not as simple as I thought. I'll revisit in another update.